planterbox/feature.py

```diff
@@ -77,6 +77,9 @@ def __init__(self, feature_path, scenarios_to_run=None, feature_text=None,
             'Feature:', '',
         ).strip()
         self.feature_doc = [doc.strip() for doc in header_text[1:]]
+        self.step_inventory = list(self.harvest_steps())
+        if not feature_path == 'foobar.feature':
```

Comment:
We don't want test limits in main code like this; I'm kind of surprised it's needed, since I think most of the "test failure" features should pass this check?